configure.sh

```diff
@@ -44,8 +44,7 @@ The following flags enable optional features (disable with --no-<option name>).
   --coverage               support for gcov coverage testing
   --profiling              support for gprof profiling
   --unit-testing           support for unit testing
-  --python2                prefer using Python 2 (also for Python bindings)
-  --python3                prefer using Python 3 (also for Python bindings)
+  --python2                prefer using Python 2 (deprecated)
```

Comment:
Could consider changing the message to something like `force use of Python2`. If you decide to change it, I think it would make sense to change the description of the CMake variable as well.

Reply:
Good point. Changed it in configure.sh and CMakeLists.txt